pkg/clusterresourceoverride/config.go

```
@@ -73,10 +79,11 @@ func Decode(reader io.Reader) (object *ClusterResourceOverride, err error) {
 
 func DecodeWithFile(path string) (object *ClusterResourceOverride, err error) {
 	reader, openErr := os.Open(path)
-	if err != nil {
+	if openErr != nil {
 		err = fmt.Errorf("unable to load file %s: %s", path, openErr)
 		return
 	}
+	defer reader.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `reader.Close` is not checked

(errcheck)